### DISCUSSION TITLE
What happens with failed tasks?

### DISCUSSION THREAD

**Discussion by Alwinator** [RealOrangeOne/django-tasks#115] [Q&A] [answered]
I am adding this package in a large enterprise django application (even if it is in alpha) and I couldn't find any documentation what happens with failed tasks:

- Will they be executed again automatically after some time? If so, how can I change the duration?
- Is it possible to retrigger failed task in the django Admin interface?

Both of those features would be a game-changer for us.

If those features are not there yet, is there a way to retrigger failed tasks from command line?

PS: I love your work so far, and this package will be a charm for django! :)

Alwin

**Comment by Alwinator** [comment]
Just found #92 which answers question 2.

**Comment by RealOrangeOne** [comment, marked as answer]
> Will they be executed again automatically after some time? 

No, currently failed tasks are left. Retries is not implemented yet #6
